src/BlazorWasmSdk/Tasks/ComputeBlazorPublishAssets.cs

```diff
@@ -195,22 +195,12 @@ public override bool Execute()
                     ITaskItem newDotNetJs = null;
                     if (aotDotNetJs != null)
                     {
-                        var itemHash = FileHasher.GetFileHash(aotDotNetJs.ItemSpec);
-                        var cacheBustedDotNetJSFileName = $"dotnet.{BundledNETCoreAppPackageVersion}.{itemHash}.js";
-
-                        var originalFileFullPath = Path.GetFullPath(aotDotNetJs.ItemSpec);
-                        var originalFileDirectory = Path.GetDirectoryName(originalFileFullPath);
-
-                        var cacheBustedDotNetJSFullPath = Path.Combine(originalFileDirectory, cacheBustedDotNetJSFileName);
-
-                        newDotNetJs = new TaskItem(cacheBustedDotNetJSFullPath, asset.CloneCustomMetadata());
```

Comment:
We didn't/shouldn't need to set the cache busted path as the definition during publish, the original aot asset item spec should be enough

Reply:
Ah okay, I believe I'd done this as I wasn't getting any published assets without it, but if this works, all the better.